README.md

```diff
@@ -8,23 +8,29 @@ All developers contributing to the project should check [Contributors Guide](./C
 
 See `./staging`
 
-- Demo market.com: <https://tonomy-id-market-com-staging.tonomy.foundation>
+- Demo market.com: <https://demo.staging.tonomy.foundation>
+- SSO website: <https://accounts.staging.tonomy.foundation>
```

Comment:
1. Remove line 13 and L29
2. SSO website: --> "Accounts website:" (also on L28)